### DISCUSSION TITLE
Cannot Find Anything in Object Browser

### DISCUSSION THREAD

**Discussion by hamzahshaq** [codefori/vscode-ibmi#851] [Q&A]
Hi, I am new in using ibm-i in vscode. 
I already connected to the server but the object browser did not show anything.
I already make sure the library and object is exist on the server..

any idea why is this happen? or where should I check?

here's capture of my vs code:
![image](https://user-images.githubusercontent.com/112717087/188196698-e396efc9-82f5-4a8b-8f0b-5afb17455f8a.png)

please help..

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after the list doesn't appear.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">
